### PR TITLE
Serve Wms From gepublish

### PR DESCRIPTION
**Pull request by hepek**
I've added an option to gepublish to pass ServeWms flag from the command line.

Don't know if this'll be useful to somebody or there's a better way to do it programmatically (without having to visit the admin panel).

### CODE REVIEW

**Comment by hepek** [comment]
I signed it!

File: earth_enterprise/src/fusion/gepublish/geserveradmin.cpp [hunk 1]
```diff
@@ -350,6 +352,7 @@ int main(int argc, char* argv[]) {
     options.flagOpt("setecdefault", ec_default_db);
     options.flagOpt("enable_poisearch", enable_poisearch);
     options.flagOpt("enable_enhancedsearch", enable_enhancedsearch);
+    options.flagOpt("serve_wms", serve_wms);
```
Comment (tst-lsavoie) [inline earth_enterprise/src/fusion/gepublish/geserveradmin.cpp]: We probably ought to have a check at some point to verify that serve_wms is not set unless publishdb is set.

**Comment by tst-lsavoie** [comment]
@hepek thanks for the contribution. From a quick glance it looks good. We'll work on getting it reviewed and tested more thoroughly.

File: earth_enterprise/src/fusion/gepublish/geserveradmin.cpp (hunk 1 above)
Comment (hepek) [reply earth_enterprise/src/fusion/gepublish/geserveradmin.cpp]: This option is only used in PublishDatabase call. See line 654.

Did you mean to check this and report to the user that an invalid option combination was set?

Comment (tst-lsavoie) [reply earth_enterprise/src/fusion/gepublish/geserveradmin.cpp]: Yes, that's the idea. I like having a check at the beginning that verifies that invalid option combinations are not used.

Comment (hepek) [reply earth_enterprise/src/fusion/gepublish/geserveradmin.cpp]: Added code at line 408.

**Comment by tst-basilhuffman** [comment]
Will test as well.

**Comment by tst-lsavoie** [comment]
@hepek I always forget this part - we like to update the release notes with every PR. The file is here: https://github.com/google/earthenterprise/blob/master/docs/geedocs/5.2.5/answer/7160006.html. Would you add this change to the new features?

**Comment by tst-basilhuffman** [comment]
Runs ok also. Good work.

**Comment by tst-lsavoie** [comment]
I'm OK with including my commits in this PR.

**Comment by tst-ccamp** [comment]
We need to retroactively add an issue for this for tracking

**Comment by tst-ccamp** [comment]
I added and linked a ticket. I didn't fill in any details though

**Pull request merged**